Buildings/Experimental/DHC/EnergyTransferStations/Cooling/Examples/DirectControlled.mo

```diff
@@ -0,0 +1,159 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling.Examples;
+model DirectControlled "Example model for direct cooling energy transfer station 
```

Comment:
In this model, `cheVal.dp` has a few 100 bar pressure drop, possibly because the flow is forced through a closed 2 way valve. This needs to be corrected.